competition-simulator.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is not the same as the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulator is based off [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.4.
+
+### Installing the simulator
+
+1. [Download the simulator](https://github.com/srobo/competition-simulator/archive/master.zip), and unzip it somewhere on your computer.
```

Comment:
I don't know what the merge policy of the target repo is, however it is a little unusual that this references `master` (a floating branch name) rather than a fixed commit hash or a tag.

Reply:
This is intentional, so that it's always pointing to the latest version.

We could use a regular tagging-based release if that's better?

Reply:
I think it's best to keep some sort of formal release process externally so it's explicit which version of the API the team is downloading at a given time (and in case a big bug makes it into Master at some point). Github has a "release" management system but regular git tags works for me too

Reply:
I think using GitHub releases (which back onto tags) is probably fine for this. I'll leave this link unpopulated until we've got said release link